django_multitenant/tests/test_utils.py

```diff
@@ -100,7 +83,21 @@ def test_tenant_filters_multi_tenant(self):
 
         unset_current_tenant()
 
-    def test_current_tenant_value_queryset(self):
+    def test_current_tenant_value_queryset_value(self):
```

Comment:
There were 2 methods with the same name. _test_current_tenant_value_queryset_
I changed their names to _test_current_tenant_value_queryset_value_ and _test_current_tenant_value_queryset_filter_
Fixed the _test_current_tenant_value_queryset_value_ since it started failing after rename